tests/Integrations/GoogleSpanner/setup-gcloud.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+export PATH=$PATH:~/.nix-profile/bin
+nix-env -i google-cloud-sdk
```

Comment:
If we need the gcloud command for debugging, we can install it using this script.
Since python3.8 cannot be installed via apt on buster, the gcloud command cannot be installed via apt.